keyboards/enviousdesign/65m/config.h

```diff
@@ -0,0 +1,157 @@
+/* Copyright 2022 Adam K (@Envious-Data)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xE739
+#define PRODUCT_ID 0xA200
+#define DEVICE_VER 0x0001
+#define MANUFACTURER "EnviousDesign"
+#define PRODUCT "EnvKB 65M"
```

Comment:
Please move this to the info.json

Reply:
does it need to be removed from config.h or is it alright to stay?

Reply:
Yeah, it would need to be removed.